mart/configs/attack/classification_eps8_pgd10_step1.yaml

```diff
@@ -1,29 +1,6 @@
 defaults:
-  - adversary
-  - perturber: default
-  - perturber/initializer: uniform_lp
-  - perturber/projector: linf_additive_range
-  - composer: additive
-  - /optimizer@optimizer: sgd
-  - gain: cross_entropy
-  - gradient_modifier: sign
-  - objective: misclassification
-  - enforcer: default
-  - enforcer/constraints: [lp, pixel_range]
-
-enforcer:
-  constraints:
-    lp:
-      eps: 8
+  - classification_eps2_pgd10_step1
```

Comment:
I think this is a bad idea. Basically adding a default that doesn't have `???` will eventually cause problems when the default changes.

Reply:
Right. I made a reusable `classification_pgd_linf.yaml` instead.